supervisor/http.py

```diff
@@ -663,7 +659,7 @@ def more(self):
             self.file.seek(-bytes_added, 2)
             bytes = self.file.read(bytes_added)
             self.sz = newsz
-            return bytes
+            return as_string(bytes)
```

Comment:
Yup, I can reproduce that error as well, and although ugly and converting between strings and bytes at multiple places, the option (using bytes everywhere) requires massive changes everywhere including in medusa. I think this may be teh best path forward. Could you update this pull request to reflect the other changes discussed with msabramo?